R/spark_compile.R

```diff
@@ -126,7 +126,7 @@ spark_compile <- function(name,
 #'
 #' @keywords internal
 #' @export
-spark_compile_package_jars <- function(package = rprojroot::find_package_root_file(),
+compile_package_jars <- function(package = rprojroot::find_package_root_file(),
```

Comment:
Okay, let's leave this as-is then.

On Fri, Aug 5, 2016 at 9:57 PM, Kevin Ushey <notifications@github.com>
wrote:

> In R/spark_compile.R
> <https://github.com/rstudio/sparklyr/pull/150#discussion_r73778751>:
>
> > @@ -126,7 +126,7 @@ spark_compile <- function(name,
> >  #'
> >  #' @keywords internal
> >  #' @export
> > -spark_compile_package_jars <- function(package = rprojroot::find_package_root_file(),
> > +compile_package_jars <- function(package = rprojroot::find_package_root_file(),
>
> pkg might be better as an argument name (for conformance with devtools),
> but I like using rprojroot since it makes it easier to call this function
> when you might have set the working directory to e.g. the subdirectory of a
> project.
>
> E.g. if you needed to write a standalone R script that lived somewhere in
> the bowels of your package sources, this would still do the right thing
> without extra 'find my package root' work from the user.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/rstudio/sparklyr/pull/150/files/dafaaa2516e9a03fac9075f1222cb9285069e69a..26772d833cb691b37a37157f0c62a54a961be082#r73778751>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAGXxyn5eszpQdxfYYuSjhh10tM7bE33ks5qc-oRgaJpZM4JeE4J>
> .
>